content/about/releases.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Releases
+description: "Releases | Node.js"
+authors: ZYSzys, nstanard,  mikeal, fhemberger, garybernhardt, piepmatz, boneskull, bjb568
+category: releases
+---
+
+Major Node.js versions enter _Current_ release status for six months, which gives library authors time to add support for them.
+After six months, odd-numbered releases (9, 11, etc.) become unsupported, and even-numbered releases (10, 12, etc.) move to _Active LTS_ status and are ready for general use.
+_LTS_ release status is "long-term support", which typically guarantees that critical bugs will be fixed for a total of 30 months.
+Production applications should only use _Active LTS_ or _Maintenance LTS_ releases.
+
+![Releases](https://raw.githubusercontent.com/nodejs/Release/master/schedule.svg?sanitize=true)
```

Comment:
Instead of fetching data of a static image, can we generate this chart ourselves by a React Chart Library using the `rawData` coming from `useReleaseData` hook. That would look cool

Reply:
that would take quite a while, I think we can go as is at MVP1 stage. Do you think it's blocker?

Reply:
It's not a blocker. I was actually worried about how the chart would look on dark mode, since it wasn't made for that. But seeing this preview link, I think it looks good for MVP1.